examples/lib/stories/effects/sequence_effect_example.dart

```diff
@@ -28,7 +28,7 @@ class SequenceEffectExample extends FlameGame {
               ScaleEffect.by(Vector2.all(1.5), duration(0.7)),
               MoveEffect.to(Vector2(400, 500), duration(0.7)),
             ],
-            alternate: true,
+            alternatePattern: AlternatePattern.excludeLast,
```

Comment:
This should be done additionally to `alternate`, so that it's not a breaking change, and also follows the same pattern as the other effects.